tools/debian/control

```diff
@@ -152,7 +140,7 @@ Provides: cockpit-shell,
           cockpit-tuned,
           cockpit-users
 Conflicts: cockpit-shell
```

Comment:
Needs a `Breaks: cockpit-dashboard` as well; a single Replaces: only means "replaces some files", but with the extra Breaks, apt will properly remove the old package on upgrades.